inc/saitam.h

```diff
@@ -1180,8 +1180,19 @@ typedef enum _sai_tam_report_type_t
 
     /**
      * @brief Report using vendor extensions
+     * deprecate and use CUSTOM_RANGE_START
```

Comment:
if one enum should be depreacated, then another should replace it maybe, but giving custom range make no sense here?